pkgs/tools/admin/ssh-import-id/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage
+, stdenv
+, fetchgit
+, requests
+, makeWrapper
+, extraHandlers ? []
+}:
+
+buildPythonPackage rec {
+  pname = "ssh-import-id";
+  version = "5.8";
+
+  src = fetchgit {
+    url = "https://git.launchpad.net/ssh-import-id";
+    rev = version;
+    sha256 = "0l9gya1hyf2qfidlmvg2cgfils1fp9rn5r8sihwvx4qfsfp5yaak";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  postFixup = ''
+    for b in $out/bin/*; do
+      # handlers require main bin, main bin requires handlers
+      wrapProgram $b \
+        --prefix PATH : "$out/bin:${stdenv.lib.makeBinPath extraHandlers}"
+    done
+  '';
```

Comment:
you can use `makeWrapperArgs`. see python nixpkgs docs.
And you should probably do `optionals extraHandlers != []` since it's default nothing.

Reply:
Can't since "$out/bin" always needs to be added.

Reply:
Added "makeWrapperArgs"